webui/react/src/services/apiBuilder.ts

```diff
@@ -1,6 +1,6 @@
 import axios, { AxiosResponse, CancelToken, Method } from 'axios';
 
-import handleError, { ErrorLevel, ErrorType, isDaError } from 'ErrorHandler';
+import handleError, { DaError, ErrorLevel, ErrorType, isDaError } from 'ErrorHandler';
 import { isAuthFailure } from 'services/api';
```

Comment:
**question:** We discussed possibly modifying `isAuthFailure`, `isLoginFailure` and `isNotFound` to handle `DaError` and `Error` types, is that still the case?

Reply:
yep, that's just more involved that simply changing this function's signature. I was considering leaving that to another step.

Reply:
let's do that if that sg  to you